scalanet/ut/src/io/iohk/scalanet/NetUtils.scala

```diff
@@ -6,15 +6,22 @@ import java.security.KeyStore
 import java.security.cert.Certificate
 import io.iohk.scalanet.peergroup.InetPeerGroupUtils
 
-import scala.collection.JavaConverters._
 import scala.util.Random
+import scala.collection.mutable.ArrayBuffer
 
 object NetUtils {
 
   val keyStore: KeyStore = loadKeyStore("keystore.p12", "password")
   val trustStore: KeyStore = loadKeyStore("truststore.p12", "password")
-  val trustedCerts: Array[Certificate] =
-    trustStore.aliases().asScala.toArray.map(alias => trustStore.getCertificate(alias))
+  val trustedCerts: Array[Certificate] = { 
+    val aliases = trustStore.aliases()
+    val buffer = ArrayBuffer[Certificate]()
+    while(aliases.hasMoreElements()) {
+      val alias = aliases.nextElement()
+      buffer.append(trustStore.getCertificate(alias))
+    }
+    buffer.toArray
+  }
```

Comment:
I think this is not a good tradeoff for something that runs once.

Reply:
I mean in general it's harder to understand this kind of Javaesque looping. It might be worth doing if the code turns out to be a hotspot, or maybe if it's hidden away to a library method, but here inline for some field that gets initialized once, I think the `.asScala` is a pretty idiomatic way to handle the conversion and the 1 line we had was easy to understand. It actually seems to return an `Iterator[String]` in this case, so perhaps moving `.toArray` to the end would avoid the interim allocation that I assume you're worried about. 

However this change has nothing to do with the PR, does it? Or has something changed in 2.13? If not, I think it's best to concentrate on the issue any given PR is doing without introducing tangential changes.

Reply:
Yes, unfortunately, all asScala and asJava methods were moved to another package, from
`scala.collection.JavaConverters` were moved to `scala.jdk.CollectionConverters`
That's the reason I've messed up the code with java-ish stuff where the java collection is needed.

An alternative solution could be
1) Suppress warnings (deprecation)
2) Remove fatal-warnings compiler directive
3) Split the sources (2.12/src and 2.13/src) 

And I don't see anything good about those options :(

Reply:
Hm, thanks for expaining. Perhaps [suppression](https://github.com/ghik/silencer#message-pattern) would be better than adding code in an ad-hoc way. @KonradStaniec wdyt?

Reply:
https://github.com/scala/scala-collection-compat/issues/208

Reply:
Perhaps the easiest option is to do as this [comment](https://github.com/scala/scala-collection-compat/issues/208#issuecomment-497710883) suggests and only use fatal warnings as part of the 2.12 options. As long as we're compiling to 2.12, that would be our primary target, no?

Reply:
> [scala/scala-collection-compat#208](https://github.com/scala/scala-collection-compat/issues/208)
I can do a workaround like this, but I was afraid that you wouldn't accept this monster :)

Reply:
Well it's kinda crazy, but it would scale better. Just make sure we have a link to that explanation :smile: